server/index.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const app = express();
+const port = 3002;
+
+app.use(express.static('public'));
```

Comment:
This is so simple! I always get into a path.join thing here with dirname but this absolute path is simpler and works. Not sure if there's any advantage of using path though? More resilient if file structure changes?

Reply:
I'm not sure! I'm barely starting to get comfortable with path.